github/pulls.go

```diff
@@ -72,6 +72,10 @@ type PullRequest struct {
 	// ActiveLockReason is populated only when LockReason is provided while locking the pull request.
 	// Possible values are: "off-topic", "too heated", "resolved", and "spam".
 	ActiveLockReason *string `json:"active_lock_reason,omitempty"`
+
+	// IsDraft identifies if the pull request is a draft or not.
+	// https://developer.github.com/changes/2019-02-14-draft-pull-requests/
+	IsDraft             *bool      `json:"is_draft,omitempty"`
```

Comment:
Based on what I see at https://developer.github.com/v3/pulls/, it seems that the field is named exactly `draft`, not `is_draft`. Do you have another source suggesting it should be `is_draft` instead?

If not, please change the JSON tag to `draft`, and also rename the Go field to `Draft`, to match.

Additionally, this is a pretty simple boolean field, so I think we can just place it in the big group of fields at the top of this struct, rather than at the bottom, separately. How do you feel about placing it between the `User` and `Merged` fields?

Reply:
You're right, I thought it was `is_draft` from looking at this one.